app/src/main/java/org/wikipedia/dataclient/page/PageLead.java

```diff
@@ -5,22 +5,256 @@
 import androidx.annotation.NonNull;
 import androidx.annotation.Nullable;
 
+import com.google.gson.annotations.JsonAdapter;
+import com.google.gson.annotations.SerializedName;
+
+import org.wikipedia.auth.AccountUtil;
+import org.wikipedia.page.GeoTypeAdapter;
+import org.wikipedia.page.Namespace;
 import org.wikipedia.page.Page;
+import org.wikipedia.page.PageProperties;
 import org.wikipedia.page.PageTitle;
+import org.wikipedia.page.Section;
+import org.wikipedia.util.UriUtil;
+
+import java.util.Collections;
+import java.util.List;
+
+import static org.wikipedia.dataclient.Service.PREFERRED_THUMB_SIZE;
 
 /**
- * Gson POJI for loading the first stage of page content.
+ * Gson POJO for loading the first stage of page content.
  */
-public interface PageLead {
-    /** Note: before using this check that #hasError is false */
-    Page toPage(PageTitle title);
+public class PageLead implements PageLeadProperties {
+    @SuppressWarnings("unused") private int ns;
```

Comment:
Would it be better if move the `@SuppressWarnings("unused")` to the top of this class?

Reply:
This class will go away when we switch over to mobile-html.

Reply:
I see.